src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+const carCosts = 40;
```

Comment:
The variable name 'carCosts' should be more descriptive to explain its purpose. Consider renaming it to something like 'DAILY_RENTAL_COST' to make it clear that it represents the daily cost of renting a car.